style.css

```diff
@@ -37,19 +37,93 @@ body {
 @media screen and (min-width: 375px) {
   /* header section */
   .header {
-    display: flex;
     height: 5vh;
-    justify-content: center;
+    position: relative;
+    z-index: 9999;
   }
 
   .header-container {
-    display: flex;
-    justify-content: end;
-    width: 90%;
+    width: 100%;
+    position: relative;
   }
 
+  .header button {
+    position: absolute;
+    z-index: 1;
+    right: 10%;
+    top: 15px;
+    background-color: transparent;
+    border: none;
+  }
+
+  /* .mobile-menu-toggle:hover nav {
+    transform: translateY(0);
+  } */
+
   .header nav {
-    display: none;
+    /* display: none; */
+    display: block;
+    position: absolute;
+    height: 100vh;
+    transform: translateY(-100%);
+    transition: transform 450ms ease-in 100ms, background-color 400ms ease-in;
+  }
+
+  .header nav[data-visible='true'] {
+    transform: translateY(0%);
+    transition: transform 450ms ease-in, background-color 200ms ease-in 450ms;
+    background-color: #e8e9f0;
+  }
+
+  .header nav ul {
```

Comment:
- **[OPTIONAL]** A UX knowen methodology is when the user opens the mobile menu the app prevents the user from scrolling down to the website so I highly recommend implementing this as well 💯 

My suggestion is to add a class to the body when the menu is opened and remove that class when the menu closed and also when menu options are clicked 👍🏼 

### CSS example:
```css
.prevent-scrolling {
    overflow: hidden
}
```

### JS example:
```javascript
// select body
const body = document.querySelector('body');
// ==> (event)hamburger clicked
body.classList.add("prevent-scrolling");
// ==> (event)close button clicked || menu option clicked
body.classList.remove("prevent-scrolling");
```